fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalTable.java

```diff
@@ -114,6 +114,11 @@ public boolean isView() {
         return false;
     }
 
+    public boolean checkInitialized() {
```

Comment:
No need this method, we can use check `objectCreated`

Reply:
i will remove this method.